ci/deploy-rpm.sh

```diff
@@ -4,13 +4,19 @@ function create_rpm_repo () {
         version=$1
         rpm_path=rpm/releases/${version}/x86_64
 
+        # removing all old rpm files except the last 5
+        files=$(ls -1 $rpm_path/trivy-* | head -n -5)
```

Comment:
It seems to remove the old five releases. Am I missing something? If I'm correct, it is problematic because it will eventually remove all old releases. This script adds one release but also removes five releases. It means reducing four releases. 10 -> 6 -> 2 -> 1 -> 1 ...

Reply:
And I'm concerned that this command fails when a new release is added like RHEL 10. `$rpm_path` doesn't exist until the dir is created.
https://github.com/aquasecurity/trivy/blob/824fd058a49e2b5fb650d5ade025e178d8a15e5d/ci/deploy-rpm.sh#L16

```
$ ls unknown_dir | head -n 5
ls: cannot access 'unknown_dir': No such file or directory
$ echo $?
0
```

`head -n 5` exits successfully, and this failure may not exit the script itself, but I want to ensure that. It would be great if you try that in your fork.